src/main/java/org/fundacionjala/sfdc/pageobjects/SalesForceContacts.java

```diff
@@ -0,0 +1,51 @@
+package org.fundacionjala.sfdc.pageobjects;
+
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.Select;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import org.fundacionjala.sfdc.commons.DriverManager;
+import java.util.Iterator;
+import java.util.List;
+
+/**
+ * SalesForceAccounts.java
+ * Class that represents Accounts page in SalesForce.
+ */
+public class SalesForceContacts {
+    private WebDriver webDriver;
+    private By contactNameTextField;
+    private By newContactButton;
+    private WebDriverWait wait;
+
+    /**
+     * Constructor.
+     * @param webDriver instance.
+     */
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the "wait" field and declare it as a local variable in the relevant methods. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1450)